plugin/trino-kafka/src/main/java/io/trino/plugin/kafka/KafkaConfig.java

```diff
@@ -174,4 +176,18 @@ public KafkaConfig setResourceConfigFiles(List<String> files)
                 .collect(toImmutableList());
         return this;
     }
+
+    @NotEmpty
+    public String getInternalFieldPrefix()
+    {
+        return internalFieldPrefix;
+    }
+
+    @Config("kafka.internal-field-prefix")
```

Comment:
nit: Since we call them internal columns in user-facing docs should this config be called `kafka.internal-column-prefix`?

Reply:
It brings some inconsistency to code, but this inconsistency already exists in definitions.
In general I agree with you.